### ISSUE TITLE
keyboard shortcuts

### ISSUE THREAD

**Issue by rafaelcastrocouto** [spicylobstergames/shotcaller-godot#95]
All player actions should be achievable with just the keyboard and it shuold be clear for the player what keys to press. just basic a11y

**Comment by erlend-sh** [comment]
We also need a complete key-map in our readme.

**Comment by jrdoughty** [comment]
do we know what commands are missing atm?

**Comment by rafaelcastrocouto** [comment]
it's easir to list whats not missing ... you should start from the menu.

**Comment by jrdoughty** [comment]
I'll have to play the game a bit more to understand it better. I'll create a list of available commands that I see, keyboard or otherwise. My understanding of MOBA games is a bit lacking to know, and the simplified controls it has keeps me from instinctively knowing what commands are there atm. I'll report back after I dive in.

**Comment by WhalesState** [comment]
we can register Shortcuts inside buttons, which will be much easier than handling them in Input event.
